site/source/tutorials/006_deploying_to_production/index.md

```diff
@@ -0,0 +1,143 @@
+Dojo 2 has a complete set of internationalization (i18n) capabilities that are beyond the scope of this tutorial. To learn more about i18n in Dojo 2, refer to the [Internationalization in Dojo 2 tutorial](../comingsoon.html).
+{% endaside %}
+
+The `--locale`, `--supportedLocales`, and `--messageBundles` arguments are used to configure the build's support for internationalization. The `--locale` argument is used to specify which language should be considered the *default*. Whenever a different locale is used that does not have a specific translation, the default locale's translation will be used. The `--supportedLocales` argument is used to specify all of the locales that the application is designed to work with. Each locale that is added here will have its rules for date formatting, currency, etc. added to the build. Finally, the `--messageBundles` argument specifies all of the message bundles, or translation files, that should be included in the build. For more information about creating internationalized applications with Dojo 2, refer to the [Internationalization](../comingsoon.html) article in the reference guide.
+
+The next two arguments that are listed when the `dojo build help` command is run are the `--element` and `--elementPrefix`. These two fields are used when exporting Dojo 2 widgets as [web components](https://en.wikipedia.org/wiki/Web_Components). They are not used when building an application.
+
+The `--debug` argument will cause the builder to generate an additional file - `profile.json`. This file contains a great deal of information about the build itself, including what modules were included and how long they took to build. A profile be inspected using WebPack's [analyze tool](https://webpack.github.io/analyse/).
+
+The final argument is `--disableLazyWidgetDetection`. A widget can be registered with the [widget registry](https://github.com/dojo/widget-core#widget-registry) to load only when required. This [lazy loading](https://en.wikipedia.org/wiki/Lazy_loading) behavior can reduce the initial load time of the application and is often useful when large, sophisticated widgets are not always required. The build system does this by creating a separate bundle for each of these lazily loaded widgets. Lazy loading can, however, cause unnecessary delays if the widgets are often needed. In those situations, adding the `--disableLazyWidgetDetection` argument will cause the the build system to include lazily loaded widgets into the main bundle. The widgets will then be immediately available for use by the application.
+
+The command line arguments give control over how the application is built and prepared for deployment. However, providing these arguments every time a build is required can be error prone. In the next section, we will learn how to create a persistent build configuration by using `.dojorc` files.
+
+{% section %}
+
+## Configuration files
+
+{% task 'Use a .dojorc file to store configuration settings.' %}
```

Comment:
Use a .dojorc file to store configuration settings -> Use .dojorc to store configuration settings